app/controllers/hyrax/workflow_actions_controller.rb

```diff
@@ -3,6 +3,13 @@ module Hyrax
   class WorkflowActionsController < ApplicationController
     DEFAULT_FORM_CLASS = Hyrax::Forms::WorkflowActionForm
 
+    ##
+    # @!attribute [r] curation_concern
+    #   @api private
+    #   @return [Hyrax::Resource]
+    attr_reader :curation_concern
+
+    load_resource class: Hyrax::Resource, instance_name: :curation_concern
     before_action :authenticate_user!
```

Comment:
we should probably be able to do `load_and_authorize_resource` but i'm wanting to only change one thing at a time, here.